### PR TITLE
Fedora source build

### PR DESCRIPTION
**Pull request by s-fairchild**
Hello All,

I would like to start contributing to tinygo-site and tinygo in the future.
I believe these changes will help future Fedora users who are new to tinygo. The majority of documentation I've come across so far is debian focused.

### CODE REVIEW

File: content/docs/guides/debugging.md [hunk 1]
```diff
@@ -55,6 +71,7 @@ Here are some debug probes that are known to work in TinyGo for at least some bo
...
+  * [ DYI Picoprobe](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html#debugging-using-another-raspberry-pi-pico): Is another DIY opensource tool for the Raspberry Pi Pico. It acts as a Serial Wire Debugger (SWD) and UART to usb converter. The Raspberry Pi Pico getting started guide Appendix A page 60 has [instructions to build](https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf) from [source](https://github.com/raspberrypi/picoprobe) or you can download and flash the [UF2 image](https://datasheets.raspberrypi.com/soft/picoprobe.uf2).
```
Comment (deadprogram) [inline content/docs/guides/debugging.md]: Note extra space.

File: content/docs/guides/debugging.md [hunk 2]
```diff
@@ -30,6 +30,22 @@ You can install the most common dependencies through the package manager:
 
     sudo apt-get install gdb-multiarch openocd
 
+### Fedora
+
+Fedora's `gdb` package comes compiled with multi arch support:
+  
+    dnf install gdb openocd
+
+To use the picprobe as a debugger you must compile openocd from source using Raspberry Pi's openocd. [Getting Started with Raspberry Pi Pico](https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf) Appendix A has build instructions we can modify:
```
Comment (deadprogram) [inline content/docs/guides/debugging.md]: Spelling?

Comment (s-fairchild) [reply content/docs/guides/debugging.md]: Thank you for pointing these two out. I've amended my commit to correct the spelling and the extra whitespace.

**Comment by s-fairchild** [comment]
@deadprogram Can you review my changes? I've rebased from the dev branch and resolved the merge conflict.

**Comment by deadprogram** [comment]
Thank you very much for the additions @s-fairchild now merging.

**Pull request merged**